config.yaml

```diff
@@ -21,12 +21,12 @@ streams:
       # type: development # do not touch; line managed by `next-devel/manage.py`
     rawhide:
       type: mechanical
-    # branched:
-    #  type: mechanical
-    # bodhi-updates:
-    #   type: mechanical
-    # bodhi-updates-testing:
-    #   type: mechanical
+    branched:
+      type: mechanical
+    bodhi-updates:
+      type: mechanical
+    bodhi-updates-testing:
+      type: mechanical
```

Comment:
Leave these commented out. We're only going to enable `branched`, not `bodhi-updates` and `bodhi-updates-testing`.